sections/Velocity_Analysis.tex

```diff
@@ -0,0 +1,38 @@
+\section{Velocity Analysis}\label{sec:velocityAnalysis}
+We have held multiple sessions of Planning Poker during the project, as seen in e.g. \ref{sec:sprintPlanningSprint3}.
+
+Doing so has allowed us to understand the velocity we work at over time. To recap: \textit{Velocity is measured by taking the sum of the estimates of each completed task in a sprint\cite{sutherlandScrumArtDoing2014}.}
+
+Once you have your velocity over multiple sprints, you can begin tracking and analyzing how the velocity changes over time. This can be useful to measure how experiments or changes to processes affect the team's velocity.
```

Comment:
```suggestion
Once velocity has been measured over multiple sprints, one can begin tracking and analyzing how the velocity changes over time. This can be useful to measure how experiments or changes to processes affect the team's velocity.
```